pages/explore-the-protocol-docs/glossary.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Glossary
+keywords: terms, glossary
+last_updated: 
+sidebar: mydoc_sidebar
+permalink: /explore-the-protocol-docs/glossary
+folder: explore-the-protocol-docs
+---
+
+# Glossary
+
+**Account** — _See Bitmark Account._
+
+**ancillary information** — Information placed on the _Bitmark blockchain_ in addition to the _Bitmark certificate_ (and potentially the _digital asset_ or _data_ itself).
+
+**asset record** — A _record_ on the _Bitmark blockchain_ that details information about a _digital property_. You _register_ an asset record.
+
+**asset** — Something of value. Usually a shorthand for _digital asset_.
+
+**authenticate (v.) — to prove that you are who you say you are, or that a _property_ is what it purports to be, as opposed to _verify_.
+
+**Bitcoin** — The technology behind the _bitcoin_ currency.
+
+**bitcoins** — The currency enabled by the _Bitcoin_ technology.
+
+**Bitmark** — The name of a company that creates systems to manage _digital property_ rights. 
+
+**Bitmark Account** — A user account at Bitmark containing a _public key_ and a _private key_. It links to _Bitmark certificates_ on the _Bitmark blockchain_ and allows a property owner to _record_, track, and trade their _digital assets_ and _data_.
+
+**Bitmark Account Number** — A public key identifying a _Bitmark Account_.
+
+**Bitmark API** — A programming interface that allows programmers to interface with the _Bitmark Property System_. It is an Open Rest API.
+
+**Bitmark blockchain** — The _blockchain_ used by Bitmark to record _Bitmark certificates_. It is both decentralized and distributed and is also open, public, and free. Also called the _Bitmark public blockchain_ or the _Bitmark public blockchain ledger_.
+
+**Bitmark certificate** — A unique identifier that provides a record of _property rights_ for _digital assets and data_. Technically, this is either an _issue record_ or a _transfer record_ that records the current ownership of a _digital property_, associated with an _asset record_ that describes what the digital property is.
+
+**Bitmark Health** — An iPhone app.
+
+**Bitmark Property System** — The _digital assets and data_ management system that consists of the _Bitmark blockchain_, the _Bitmark Account_, and other tools.
+

```

Comment:
I noted "Bitmark Protocol" as a new phrase in these docs, which should get a definition here.